install_config/aggregate_logging.adoc

```diff
@@ -724,6 +724,12 @@ named to match the `openshift_logging_es_pvc_prefix` setting, which defaults to
 exists already, it is used; if not, and `openshift_logging_es_pvc_size` has been
 specified, it is created with a request for that size.
 
+If using xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#install-config-persistent-storage-dynamically-provisioning-pvs[dynamically provisioned PVs], the {product-title} logging installer creates PVCs that use the default storage class or the one specified with the `openshift_logging_elasticsearch_pvc_storage_class_name` parameter.
+
+If using NFS storage, the {product-title} installer creates the persistent volumes, based on the `openshift_logging_storage_*` parameters 
```

Comment:
I think we should mention two points
[1]The logging deploy playbook don't trigger nfs storage allocation. you must run the cluster deployment playbook  /usr/share/ansible/openshift-ansible/playbooks/deploy_cluster.yml
[2] For nfs is unssupported, you have to set openshift_enable_unsupported_configurations=true
 
1) this variable must be set openshift_enable_unsupported_configurations=true.

Reply:
@jcantrill @anpingli I am moving these suggested changes to a new PR so we can merge this PR independently. 
https://github.com/openshift/openshift-docs/pull/12834